### PR TITLE
Add negative feedback option to Rocchio 

### PR DESCRIPTION
**Pull request by yuki617**
* Add negative label to Rocchio reranker
* Add tests for Rocchio with negative feedback

### CODE REVIEW

File: pyserini/search/lucene/__main__.py [hunk 1]
```diff
@@ -88,6 +88,7 @@ def define_search_args(parser):
 
     parser.add_argument('--rm3', action='store_true', help="Use RM3")
     parser.add_argument('--rocchio', action='store_true', help="Use Rocchio")
+    parser.add_argument('--usenegative', action='store_true', help="Use nonrelevant labels in Rocchio")
```
Comment (lintool) [inline pyserini/search/lucene/__main__.py:91]: change to `--use-negative` please.

File: pyserini/search/lucene/__main__.py [hunk 2]
```diff
@@ -182,7 +183,10 @@ def define_search_args(parser):
     
     if args.rocchio:
         search_rankers.append('rocchio')
-        searcher.set_rocchio()
+        if args.usenegative:
+            searcher.set_rocchio(rocchio_gamma=0.15, rocchio_use_negative=True)
```
Comment (lintool) [inline pyserini/search/lucene/__main__.py:187]: Do we need `rocchio_` prefix here? Method already disambiguates... right?

**Review by lintool — CHANGES_REQUESTED** [review]
Minor comments? Thoughts?

File: pyserini/search/lucene/__main__.py (hunk 1 above)
Comment (yuki617) [reply pyserini/search/lucene/__main__.py:91]: Will --rocchio-use-negative be better? To be more specific for rocchio reranking.
